### PR TITLE
Update libs

### PR DESCRIPTION
**Pull request by hannesa2**
bumped ddmlib version in hopes it would solve a ddmlib crash
#523

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -36,16 +36,16 @@ buildscript {
...
+              'espressoContrib': 'com.android.support.test.espresso:espresso-contrib:3.0.1',
+              'runner': 'com.android.support.test:runner:1.0.1',
+              'rules': 'com.android.support.test:rules:1.0.1',
           ],
       ],
   ]
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.0.0-beta7'
+    classpath 'com.android.tools.build:gradle:3.1.0-beta3'
```
Comment (JakeWharton) [inline build.gradle]: Let's stick to 3.0.1. Is there any reason to use a beta here?

File: .travis.yml [hunk 2]
```diff
@@ -4,13 +4,14 @@ android:
...
     - platform-tools
-    - build-tools-26.0.2
+    - build-tools-27.0.3
 
 jdk:
   - oraclejdk8
 
 before_install:
   # Install SDK license so Android Gradle plugin can install deps.
+  - yes | sdkmanager "platforms;android-27"
```
Comment (JakeWharton) [inline .travis.yml]: Delete. AGP will install this itself.

File: .travis.yml [hunk 3]
```diff
@@ -4,13 +4,14 @@ android:
   components:
     - tools
     - platform-tools
-    - build-tools-26.0.2
+    - build-tools-27.0.3
```
Comment (JakeWharton) [inline .travis.yml]: Delete this line. AGP will install build tools itself.

File: build.gradle [hunk 4]
```diff
@@ -1,8 +1,8 @@
 buildscript {
   ext.versions = [
       'minSdk': 9,
-      'compileSdk': 25,
-      'buildTools': '26.0.2',
+      'compileSdk': 27,
+      'buildTools': '27.0.3',
```
Comment (JakeWharton) [inline build.gradle]: You can delete this property and all references to it. AGP will install what it needs itself.

File: .travis.yml [hunk 5]
```diff
@@ -4,13 +4,14 @@ android:
...
 
 jdk:
   - oraclejdk8
 
 before_install:
   # Install SDK license so Android Gradle plugin can install deps.
+  - yes | sdkmanager "platforms;android-27"
   - mkdir "$ANDROID_HOME/licenses" || true
   - echo "8933bad161af4178b1185d1a37fbf41ea5269c55" > "$ANDROID_HOME/licenses/android-sdk-license"
```
Comment (JakeWharton) [inline .travis.yml]: This is the old hash. Update to `d56f5187479451eabf01fb78af6dfcb131a6481e`.

File: build.gradle [hunk 6]
```diff
@@ -4,7 +4,7 @@ buildscript {
       'compileSdk': 27,
       'buildTools': '27.0.3',
 
-      'supportLibrary': '27.0.2',
+      'supportLibrary': '25.3.1',
```
Comment (originx) [inline build.gradle]: cant we just bump the minSDK to 14 here and align it with suppLib? @JakeWharton I guess its time no? :D

Comment (JakeWharton) [reply build.gradle]: Yes.

File: build.gradle (hunk 4 above)
Comment (hannesa2) [reply build.gradle]: The main goal of my PR is to update this
`ddmlib': 'com.android.tools.ddms:ddmlib:26.0.1`
Ok, I update some more libs, but for me it's hard to remove some variables, which I didn't introduced, and it's usages, which the inventor doesn't like anymore.

It's more effective if you change it, you know exactly what you want and how should it look like

**Pull request merged**